projects/core/src/lib/functions/array.ts

```diff
@@ -49,3 +51,40 @@ export function findIndex<T>(items: T[], valueToFind: any, property?: string): n
 export function pluck<T = any>(items: any[], property: string): T[] {
 	return (items instanceof Array) ? items.map(i => getValue(i, property)) : [];
 }
+
+export function compareItems<T = any>(item1: T, item2: T, ...compareProperties: (string | CompareProperty)[]): -1 | 0 | 1 {
```

Comment:
Incorporate the ascending property if given a CompareProperty instead of a string